SPECS/libvirt/libvirt.spec

```diff
@@ -1,654 +1,1970 @@
+Obsoletes: libvirt-daemon-driver-uml <= 5.0.0
+Obsoletes: libvirt-daemon-uml <= 5.0.0
+%if %{with_vbox}
+Requires: libvirt-daemon-driver-vbox = %{version}-%{release}
+%endif
+Requires: libvirt-daemon-driver-nwfilter = %{version}-%{release}
+Requires: libvirt-daemon-driver-interface = %{version}-%{release}
+Requires: libvirt-daemon-driver-secret = %{version}-%{release}
+Requires: libvirt-daemon-driver-storage = %{version}-%{release}
+Requires: libvirt-daemon-driver-network = %{version}-%{release}
+Requires: libvirt-daemon-driver-nodedev = %{version}-%{release}
+Requires: libvirt-client = %{version}-%{release}
+Requires: libvirt-libs = %{version}-%{release}
+
+# All build-time requirements. Run-time requirements are
+# listed against each sub-RPM
+BuildRequires: python3-docutils
+BuildRequires: meson >= 0.56.0
+BuildRequires: ninja-build
+# bfjelds: NEW
+BuildRequires: git
+# bfjelds: NEW
+BuildRequires: perl-interpreter
+# bfjelds: NEW
+BuildRequires: python3
+# bfjelds: NEW
+BuildRequires: python3-pytest
+# For xmllint
+BuildRequires: libxml2
+# For xsltproc
+BuildRequires: libxslt
+BuildRequires: gettext
+# bfjelds: NEW
+BuildRequires: systemd-rpm-macros
+# Fedora build root suckage
+# bfjelds: NEW
+BuildRequires: gawk
+%if %{with_native}
+# bfjelds: NEW
+BuildRequires: gcc
+    %if %{with_libxl}
+BuildRequires: xen-devel
+    %endif
+# bfjelds: NEW
+BuildRequires: glib2-devel >= 2.56
+BuildRequires: libxml2-devel
+BuildRequires: readline-devel
+BuildRequires: bash-completion >= 2.0
+# bfjelds: NEW
+BuildRequires: libtasn1-devel
+BuildRequires: gnutls-devel
+BuildRequires: libattr-devel
+# For pool-build probing for existing pools
+# bfjelds: NEW
+BuildRequires: libblkid-devel >= 2.17
+# for augparse, optionally used in testing
+BuildRequires: augeas
+BuildRequires: systemd-devel >= 185
+BuildRequires: libpciaccess-devel >= 0.10.9
+BuildRequires: yajl-devel
+    %if %{with_sanlock}
+BuildRequires: sanlock-devel >= 2.4
+    %endif
+BuildRequires: libpcap-devel >= 1.5.0
+BuildRequires: libnl3-devel
+BuildRequires: libselinux-devel
+BuildRequires: iptables
+BuildRequires: ebtables
+# For modprobe
+# bfjelds: NEW
+BuildRequires: kmod
+BuildRequires: cyrus-sasl-devel
+BuildRequires: polkit >= 0.112
+# For mount/umount in FS driver
+BuildRequires: util-linux
+    %if %{with_qemu}
+# For managing ACLs
+BuildRequires: libacl-devel
+# From QEMU RPMs, used by virstoragetest
+%if 0%{?mariner}
+BuildRequires: qemu-img
+%else
+BuildRequires: /usr/bin/qemu-img
+%endif
+    %endif
+# nbdkit support requires libnbd
+    %if %{with_nbdkit}
+BuildRequires: libnbd-devel
+    %endif
+# For LVM drivers
+BuildRequires: lvm2
+# For pool type=iscsi
+BuildRequires: iscsi-initiator-utils
+    %if %{with_storage_iscsi_direct}
+# For pool type=iscsi-direct
+BuildRequires: libiscsi-devel
+    %endif
+# For disk driver
+# bfjelds: NEW
+BuildRequires: parted-devel
+# For Multipath support
+BuildRequires: device-mapper-devel
+    %if %{with_storage_rbd}
+%if 0%{?mariner}
```

Comment:
i think this is right  (link to versioned deps), it matches previous spec dependencies